Wrappers/Python/test/test_Gradient.py

```diff
@@ -240,3 +239,193 @@ def test_Gradient_linearity(self):
         grad = Gradient(ig, bnd_cond='Periodic', correlation='SpaceChannels', backend='numpy')
         self.assertTrue(LinearOperator.dot_test(grad))
         
+    def test_Gradient_c_numpy_voxel(self):
+        
+        numpy.random.seed(10)
+        
+        print("Test Gradient for 2D Geometry, ")
+        
+        ny, nx, nz = 3, 4, 5
+        ig = ImageGeometry(voxel_num_y = ny, voxel_num_x = nx, voxel_size_x=0.1, voxel_size_y=0.5) 
+            
+        GD_C = Gradient(ig, backend = 'c')
+        GD_numpy = Gradient(ig, backend = 'numpy')
+                       
+        print("Check Gradient_C, Gradient_numpy norms")
+        Gradient_C_norm = GD_C.norm()
+        Gradient_numpy_norm = GD_numpy.norm()    
+        numpy.testing.assert_approx_equal(Gradient_C_norm, Gradient_numpy_norm, significant = 1) 
```

Comment:
I would consider using numpy.testing.assert_all_close() so you can set the relative and absolute tolerance. It's a bit more meaningful than approx_equal.

Reply:
done